plugins.d/pip.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Add a check to see if `pip` is available, it's not available by default on OS X.